packages/analytics/src/interface.ts

```diff
@@ -0,0 +1,61 @@
+import { Types } from 'mongoose'
+
+export interface User {
+  username: string
+}
+export interface SessionDocument {
+  _id: Types.ObjectId
+  user: User
+  quest: string
+  status: string
+  createdAt: Date
+  finishedAt: Date
+}
+export interface CommandObject {
+  command: string
+  pwd: string
+  stage: string | null | undefined
```

Comment:
```suggestion
  stage?: string | null
```
Use `?:` for optional field